MinimedKit/Extensions/NSDateFormatter.swift

```diff
@@ -20,4 +20,14 @@ extension DateFormatter {
 
         return formatter
     }
+    
```

Comment:
This is the same method copied from above, with the exception of the fractional seconds in the format. See notes further in PR regarding this.

Reply:
I think you can do this: 
```
let formatter = ISO8601DateFormatter()
formatter.formatOptions =  [.withInternetDateTime, .withFractionalSeconds]
```

Reply:
I updated this and the other method in this class to match since there was a TODO for the same. Since these are class methods on DateFormatter, I updated it to be an extension on ISO8601DateFormatter.